releasenotes/notes/procfs-path-to-host-proc-324c6ed8aa08a100.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Added the "procfs_path" configuration to `/host/proc` when containerized.
```

Comment:
What do you think about this rephasing:
```
Set the default "procfs_path" configuration to `/host/proc` when containerized
to allow the network check to collect the host's network metrics. This can be
overriden with the the DD_PROCFS_PATH envvar.
```

Reply:
Definitely